shared/src/lib.rs

```diff
@@ -14,6 +14,7 @@ pub const VERSION_STR: &str = "0.5.0";
 pub struct ClientInfoPrivate {
     pub name: String,
     pub secret: String,
+    pub steamid64: String,
```

Comment:
Maybe could be `Some(String)` instead so in the Lua-side it's `nil` if we couldn't get a SteamID. Makes using it a bit more intuitive.
```lua
if conn.getSteamID() then
  print("Connection has a SteamID")
else
  print("Connection does _not_ have a SteamID")
end
```
Otherwise you have to check for a empty string.
```lua
if conn.getSteamID() == "" then
--[[ …
```

Reply:
I suppose so yeah.
Also, I tried changing ateamid in game yesterday and it's readonly

Reply:
@vulcan-dev You could redefine `Steam` with a table that holds a reference to the original `Steam` but intercepts `getSteamID()` in some fashion.

i.e. store a reference to the original `Steam`, define a new table over the `Steam` variable, and declare a meta table with `__index` to pass everything _except_ `getSteamID` to a function that supplies something else.